convert2rhel/pkghandler.py

```diff
@@ -677,7 +677,7 @@ def _get_package_repositories(pkgs):
 
     output, retcode = utils.run_subprocess(
         ["repoquery", "--quiet", "-q"] + pkgs + ["--qf", query_format],
-        print_cmd=False,
+        print_cmd=True,
```

Comment:
Isn't the print of the command some leftover from testing?

Reply:
Michal said we might as well always print it

Reply:
Yeah, it's a debug print that might help with finding problems.